src/heroes/sombra/stealth.opy

```diff
@@ -50,8 +50,8 @@ def enterStealth():
     wait(OW2_SOMBRA_STEALTH_COOLDOWN)
     wait(OW2_SOMBRA_STEALTH_CAST_TIME)
     hideStealthHud()
+    exitStealth()
     eventPlayer.setInvisibility(Invis.NONE)
-    eventPlayer.setMoveSpeed(percent(OW1_SOMBRA_STEALTH_MOVEMENT_BONUS/OW2_SOMBRA_STEALTH_MOVEMENT_BONUS))
```

Comment:
You need this line for the movement speed bonus during "fake" stealth

Reply:
i remove it due to it sometimes when youre not in stealth it still applys the move speed. So hence thats why i remove it the default speed still applys when i removed it which is 5.50 and 4.95. ill try to find a way to keep that in while not completly removing it.

Reply:
and also just in general the passive stealth barely applys (like half a second ish so i dont think it matters too much but yeah i get your point, the speed still kinda matters when getting out of stealth.)